fdbclient/include/fdbclient/SimKmsVault.cpp

```diff
@@ -154,10 +154,8 @@ uint32_t maxSimKeys() {
 void forceLinkSimKmsVaultTests() {}
 
 TEST_CASE("/simKmsVault") {
-	auto& g_knobs = IKnobCollection::getMutableGlobalKnobCollection();
```

Comment:
Should we instead destroy and recreate the vault so that each unit test maintains it's own state? It seems weird to me that one unit test would depend on the results of another

Reply:
For simulation it is a Flow singleton process, I don't think we need to do any such thing or feels more complicated than the test requirement. Also, the assertion to some extent is more strict than needed, retaining >0 assert is more than sufficient in all cases.